### PR TITLE
30 api for image gallery caroussel from exampledatajson - Mohamed Abdalla

### LINKED ISSUE #30: API for image gallery caroussel from exampleData.json

### PR DESCRIPTION
**Pull request by Mohamed-Abdalla-1**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

## Related to

api for image gallery caroussel from exampledatajson #30 

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/pages/Home.js [hunk 1]
```diff
@@ -14,20 +14,37 @@ export function Home() {
...
+	// 			}
+	// 			return res.json();
+	// 		})
+	// 		.then((body) => {
+	// 			setMessage(body.message);
+	// 		})
+	// 		.catch((err) => {
+	// 			console.error(err);
+	// 		});
+	// }, []);
```
Comment (oliverlloyd) [inline client/src/pages/Home.js]: Shall we delete this?

File: client/src/components/ImageCard.js [hunk 2]
```diff
@@ -23,12 +23,12 @@ const ImageCard = ({ image, isLogin }) => {
 
 	return (
 		<div className="image-card">
-			<img width="400" height="250" src={`/${image.sr}`} alt="" />
+			<img width="400" height="250" src={image.src} alt="" />
```
Comment (oliverlloyd) [inline client/src/components/ImageCard.js:26]: Nice spot

File: client/src/components/Gallery.js [hunk 3]
```diff
@@ -1,12 +1,21 @@
...
+		<>
+			{message ? (
+				<h1 className="message" data-qa="message">
+					{message}
+				</h1>
+			) : (
+				<div className="gallery-section">
+					{images.map((image) => (
+						<ImageCard key={image.id} image={image} isLogin={isLogin} />
+					))}
```
Comment (oliverlloyd) [inline client/src/components/Gallery.js:15]: Textbook, perfect

**Review by oliverlloyd — APPROVED** [review]
Great

File: client/src/components/ImageCard.js (hunk 2 above)
Comment (Mohamed-Abdalla-1) [reply client/src/components/ImageCard.js:26]: Thanks

File: client/src/pages/Home.js (hunk 1 above)
Comment (Mohamed-Abdalla-1) [reply client/src/pages/Home.js]: I fixed this in this commit:
https://github.com/Innapoliakova/set/pull/31/commits/d848baaba3abdf8a1872c43fad29605939728c3e

**Pull request merged**